src/templates/service.ts

```diff
@@ -4,7 +4,7 @@ export const ServiceYAMLTemplates = `
 apiVersion: ${ServiceModel.apiVersion}
 kind: ${ServiceModel.kind}
 metadata:
-  name: exampleasd
+  name: example
```

Comment:
are u sure about this name? for a service? its a default yaml or something?

Reply:
yes this is the example yaml.
when you want to create a new service , this is what we suggest